book/website/communication/presentations/presentations-remote.md

```diff
@@ -37,26 +37,26 @@ Presenting a talk through a digital medium involves tackling some technical chal
 * The simplest way to record the contents of your talk, including your video, voice and presentation, is to record it using a videoconference service (like Zoom) or recording software like [OBS](https://obsproject.com). See [this guide](https://software.ac.uk/fellowship-programme/2019/application-video-guide) by the Software Sustainability Institute on how to configure OBS for screen recording.
 * If you are recording your talk to present it asynchronously you can use platforms like [Amara](https://amara.org) to generate your subtitles from the video.
 
-## What to Expect from Speaking Invitation
+## What to Expect from Speaking Invitations
 
-If you have published a research piece (like a scientific paper, software or blog), or if you are known for your expertise in your field, you are likely to get invited to give talks at public forums, panels and conference sessions.
+If you have published a research piece (like a scientific paper, software or blog), or if you are known for having expertise in your field, you are likely to get invited to give talks at events such as public forums, panels and conference sessions.
 It could also be something you do upon request, for example, for graduate schools where young researchers can learn from your research work or career path.
 It is beneficial for your career progression as more people get to know you, learn about your work and establish collaborations.
 By giving talks at different institutions and events you can receive timely feedback on your progress and improve the quality and impact of your research.
-For event organisers, inviting experts as speakers translates into the popularity of their events where participants have the opportunities to learn from the experts and exchange ideas on topics of common interests.
+For event organisers, inviting experts as speakers can increase the popularity of their events because the participants have opportunities to learn from experts and exchange ideas on topics of common interest.
 
 ### Honoraria for Delivering a Talk
 
-> Honoraria recognises speaker's work by recompensing for their time and expertise invested in developing and delivering a talk.
+> Honoraria recognises the speaker's work by recompensating them for the time and expertise they invested in developing and delivering a talk.
```

Comment:
Emma, I wanted to use recompense (n, compensation or reward given for loss or harm suffered or effort made) in this context.

Reply:
thats fine - change it back!